pkg/lib/authenticationflow/declarative/generate_config_login_flow_test.go

```diff
@@ -68,12 +70,18 @@ steps:
           - authentication: device_token
         - type: change_password
           target_step: authenticate_primary_email
+- name: authenticate_amr_constraints
+  type: authenticate
```

Comment:
Change it to

```
- type: trigger_event_pre_authenticated
- type: enforce_constraints_amr
- type: change_password
- type: check_account_status
- type: terminate_other_sessions
-  
```

Reply:
https://linear.app/authgear/issue/DEV-2820/implicitly-trigger-pre-authenticated-after-all-authentication-steps

We will not add these steps.

Will handle the change in another pr.